docs/_partials/create-invitation.mdx

```diff
@@ -0,0 +1,123 @@
+<Tabs items={["Next.js", "Astro", "Express", "React Router", "Tanstack React Start"]}>
+  <Tab>
+    ```ts {{ filename: 'app/api/example/route.ts' }}
+    import { clerkClient } from '@clerk/nextjs/server'
+    import { NextResponse } from 'next/server'
+
+    export async function GET() {
+      try {
+        const client = await clerkClient()
+        const invitation = await client.invitations.createInvitation({
+          emailAddress: 'invite@example.com',
+          redirectUrl: 'https://www.example.com/my-sign-up',
+          publicMetadata: {
+            example: 'metadata',
+            example_nested: {
+              nested: 'metadata',
+            },
+          },
+        })
+        return NextResponse.json({ message: 'Invitation created', invitation })
+      } catch (error) {
+        console.log(error)
+        return NextResponse.json({ error: 'Error creating invitation' })
+      }
+    }
+    ```
+  </Tab>
+
+  <Tab>
+    ```tsx {{ filename: 'src/api/example.ts' }}
+    import type { APIRoute } from 'astro'
+    import { clerkClient } from '@clerk/astro/server'
+
+    export const GET: APIRoute = async (context) => {
+      await clerkClient(context).invitations.createInvitation({
+        emailAddress: 'invite@example.com',
+        redirectUrl: 'https://www.example.com/my-sign-up',
+        publicMetadata: {
+          example: 'metadata',
+          example_nested: {
+            nested: 'metadata',
+          },
+        },
+      })
+
+      return new Response(JSON.stringify({ success: true }), { status: 200 })
+    }
+    ```
+  </Tab>
+
+  <Tab>
+    ```ts {{ filename: 'public.ts' }}
+    import { getAuth, clerkClient } from '@clerk/express'
+
+    app.post('/createUser', async (req, res) => {
```

Comment:
This is the only `POST` in all the examples, which begs the question: 

1. Should it be a `GET` for consistency
2. Technically, wouldn't using a `POST` in all of these be a more accurate method for performing an "create" action?